lib/plug/amqp/consumer_producer.ex

```diff
@@ -394,7 +391,7 @@ defmodule Plug.AMQP.ConsumerProducer do
   #
 
   @impl true
-  def handle_cast(
+  def handle_info(
```

Comment:
If we change this to `handle_info` we should move this to it proper place (next to the other `handle_info`) and also move the above comment and probably remove the `@impl true`. Also from:

```
  #
  # Server - Handle Cast
  #
```

To something like:

```
  #
  # Server - Handle Info - Send Resp (or client calls, or whatever)
  #
```